test/zdtm/static/Makefile

```diff
@@ -240,6 +241,7 @@ TST_NOFILE	:=				\
 		zombie_leader			\
 		sigtrap				\
 		sigtrap01			\
+		change_mnt_context		\
```

Comment:
This change looks unrelated and it was already introduced with https://github.com/checkpoint-restore/criu/commit/cbaa59976dbf9db618a7bbf3e9f175f40bba72bc.